express/server.js

```diff
@@ -146,29 +149,37 @@ app.post('/checkout', async(req, res) => {
   const form = order['form'];
   let inventory = [];
   try {
+    const transaction = Sentry.getCurrentHub()
+      .getScope()
+      .getTransaction();
+    
     // Get Inventory
-    let transaction = Sentry.startTransaction({ name: '/checkout.get_inventory' });
-    let span = transaction.startChild({ op: '/checkout.get_inventory'});
+    let span = transaction.startChild({
+      op: "function",
+      description: "getInventory",
+    });
     inventory = await DB.getInventory(cart);
     console.log("> /checkout inventory", inventory);
-    span.finish();
-    transaction.finish();
 
+    span.finish();
+    
     // Process Order
-    transaction = Sentry.startTransaction({ name: 'process order' });
-    span = transaction.startChild({ op: 'process_order', description: 'function' });
+    let span2 = transaction.startChild({
```

Comment:
If you don't want to reuse the `span` variable as was previously being done, I recommend we name each span according to what they measure instead of using `span2`, which is not very descriptive.

i.e. `getInventorySpan` and `processOrderSpan` instead of `span` and `span2`.

Reply:
The wordiness makes me lose track of the 'span' variables but they should be named. I might do spanGetInventory spanProcessOrder if that's okay. Helps me find it faster if I'm skimming the code. Maybe it's a personal preference for naming.